saleor/graphql/core/mutations.py

```diff
@@ -157,19 +174,22 @@ def construct_instance(cls, instance, cleaned_data):
         return instance
 
     @classmethod
-    def user_is_allowed(cls, user):
+    def check_permissions(cls, user):
         """Determine whether user has rights to perform this mutation.
 
         Default implementation assumes that user is allowed to perform any
-        mutation. By overriding this method, you can restrict access to it.
-        `user` is the User instance associated with the request and `input` is
+        mutation. By overriding this method or defining required permissions
+        in the meta-class, you can restrict access to it. `user` is
+        the User instance associated with the request and `input` is
```

Comment:
The "input" part is outdated as we've changed the signature of this method recently.